lib/access_check_filter.js

```diff
@@ -64,6 +66,9 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 }
 
                 return contentPromise.then((content) => redirect(content, location, options));
+            } else if (content.status === 302) {
```

Comment:
We return 302 when content's not in storage && it's a redirect. For `?redirect=false` and some other conditions we don't what 302 but want 200 instead